### DISCUSSION TITLE
Stream Deck Plus support

### DISCUSSION THREAD

**Discussion by cros13** [streamdeck-linux-gui/streamdeck-linux-gui#38] [Ideas / Feature Requests]
My OG streamdeck is working OK. But seems to be no support for Streamdeck plus (USB ID 0fd9:0084).
Is this an upstream issue in the python-elgato-streamdeck library?

**Comment by coolapso** [comment]
hi, yes. 

Seems that it is not yet suppoerted on the python-elgato-streamdeck library: 

https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/issues/10
